packages/core/src/overlay/OverlayHeader.css

```diff
@@ -0,0 +1,31 @@
+.saltOverlayHeader {
+  padding: var(--salt-spacing-100);
+  width: 100%;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: stretch;
+  gap: var(--salt-spacing-100);
+  box-sizing: border-box;
+}
+
+.saltOverlayHeader-container {
```

Comment:
https://github.com/jpmorganchase/salt-ds/pull/4138#discussion_r1766838600 

changed "body" for "container" for consistency.